types/deploy.go

```diff
@@ -1,8 +1,10 @@
 package types
 
 const (
-	LonghornManagerDaemonSetName = "longhorn-manager"
-	LonghornUIDeploymentName     = "longhorn-ui"
+	LonghornManagerDaemonSetName            = "longhorn-manager"
+	LonghornUIDeploymentName                = "longhorn-ui"
+	LonghornAdmissionWebhookDeploymentName  = "longhorn-admission-webhook"
+	LonghornConversionWebhookDeploymentName = "longhorn-conversion-webhook"
```

Comment:
There are no deployments for webhooks anymore in 1.5, so adding these const here are misleading. 

Suggest just adding them to the upgrade.go, so they can be specific there.

Reply:
Great point. Thank you! Fixed